### PR TITLE
Update the Commenting Style in the DPDK backend

### LINKED ISSUE #4567: Format Comments Consistently in P4C.
This issue aims to address the lack of consistency in comment formatting across the core parts of the P4C compiler. Following a discussion in #4563, it has been suggested to adopt a unified commenting style: using triple slashes (///) for documenting functions and classes, and double slashes (//) for comments within code. This approach aligns with the recommendation provided in the Doxygen documentation.

### PR DESCRIPTION
**Pull request by sumit7754**
fixes #4567 

I have made the following changes in DPDK(backend):

Used triple slashes (///) for commenting functions and classes.
Used double slashes (//) for comments within the code.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -115,7 +115,10 @@ overview of the compiler goals and architecture.
 ## Documentation Comments Style Guide 
 - Use triple slashes `///` for documenting functions and classes in files.
 - Double slashes `//` should be used for "internal" comments within functions.
+- Double slashes `//` should be used for inline comment.
 - For rare occasions such as adding comments to multi-line macros, you may use `/* ... */` style comments.
+- There should be no space at the end of the comment.
+- First latter of comment should be Capital letter.
```
Comment (fruffy) [inline docs/README.md:121]:
```suggestion
- First letter of the comment should be a capital letter.
```

Comment (fruffy) [reply docs/README.md:121]: Also add that each comment should end with a fullstop/period. Please also do this for the comments in this PR.

**Review by fruffy — CHANGES_REQUESTED** [review]
Wonderful, thank you. Some suggestions.

File: docs/README.md (hunk 1 above)
Comment (sumit7754) [reply docs/README.md]: Any other suggestion that we can implement?

File: docs/README.md [hunk 2]
```diff
@@ -115,7 +115,11 @@ overview of the compiler goals and architecture.
 ## Documentation Comments Style Guide 
 - Use triple slashes `///` for documenting functions and classes in files.
 - Double slashes `//` should be used for "internal" comments within functions.
+- Double slashes `//` should be used for inline comment.
 - For rare occasions such as adding comments to multi-line macros, you may use `/* ... */` style comments.
+- There should be no space at the end of the comment.
+- First letter of the comment should be a capital letter.
+- Each comment should be end with a fullstop/period.
```
Comment (fruffy) [inline docs/README.md:122]:
```suggestion
- Each comment should end with a period.
```

**Review by fruffy — APPROVED** [review]

**Pull request merged**